perfkitbenchmarker/packages/fio.py

```diff
@@ -22,9 +21,12 @@
 GIT_REPO = 'git://git.kernel.dk/fio.git'
 GIT_TAG = 'fio-2.1.14'
 FIO_PATH = FIO_DIR + '/fio'
+FIO_CMD_PREFIX = '%s --output-format=json ' % FIO_PATH
```

Comment:
Recommend removing the trailing space here, and not relying on it in other modules.